.ts-for-gir.copy-all.rc.js

```diff
@@ -0,0 +1,17 @@
+export default {
+    outdir: './girs',
+    modules: ['*'],
+    girDirectories: [
+        '/usr/local/share/gir-1.0',
+        '/usr/share/gir-1.0',
+        '/usr/share/*/gir-1.0',
+        '/usr/share/gnome-shell',
+        '/usr/share/gnome-shell/gir-1.0',
+        '/usr/lib64/mutter-*',
+        '/usr/lib/mutter-*',
+        '/usr/lib/x86_64-linux-gnu/mutter-*',
+
+        './vala-girs/gir-1.0',
```

Comment:
Was one of the benefits of this to get rid of `vala-girs`? Are there still GIRs we aren't hosting here?

Reply:
we should probably have all the girs ourselves now and could remove the submodule 👍